.github/workflows/tests.yml

```diff
@@ -140,7 +140,14 @@ jobs:
           - "1.0.11"
           - "1.1.9"
           - "1.2.9"
-          - "1.3.7"
+          - "1.3.10"
+          - "1.4.7"
+          - "1.5.7"
+          - "1.6.6"
+          - "1.7.5"
+          - "1.8.5"
+          - "1.9.8"
+          - "1.10.4"
```

Comment:
Would it be possible to just specify the minor version, like `1.10`, so we don't have to update it so often?

Reply:
In theory yes but it would require some more significant refactoring in other places because right now the underlying code expects exact versions rather than constraints https://github.com/hashicorp/terraform-exec/blob/4497f9edf36dcdbbf5fc982d1d951c9509d865fd/tfexec/internal/testutil/tfcache.go#L70-L73